src/components/product-menu/product-menu.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PopoverTrigger from '@mapbox/react-popover-trigger';
+import Icon from '@mapbox/react-icon';
+import { ProductMenuDropdown } from '../product-menu-dropdown/product-menu-dropdown';
+// You could import this list from, say, a data selector if you like
+// On Mapbox docs pages, categories data is imported from a central location
+// to keep menus in sync across the site.
+import { ProductMenuItems } from '../../data/product-menu-items';
+
+const popoverProps = {
+  placement: 'bottom',
+  themePopover:
+    'round shadow-darken25 viewport-almost-but-not-always scroll-auto scroll-styled'
```

Comment:
`viewport-almost-but-not-always` is not in Assembly, right?

Reply:
per chat with @colleenmcginnis, we're going to move all of the docs-specific UI out of copypasta limbo and into this repo for docs-specific-components to reference.